<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Purisa</family>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string extracomment="Transform your photos">ARInk</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/frontend/resources/pak2/003-photos.png</normaloff>:/frontend/resources/pak2/003-photos.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="font">
     <font>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="fileOpenAct"/>
    <addaction name="saveImgAct"/>
    <addaction name="saveAsAct"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="zoomInAct"/>
    <addaction name="zoomOutAct"/>
    <addaction name="fitToWindowAct"/>
    <addaction name="normalSizeAct"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="fileOpenAct"/>
   <addaction name="saveImgAct"/>
   <addaction name="separator"/>
   <addaction name="zoomInAct"/>
   <addaction name="zoomOutAct"/>
   <addaction name="fitToWindowAct"/>
   <addaction name="separator"/>
   <addaction name="originalAct"/>
   <addaction name="grayscaleAct"/>
   <addaction name="colorizeAct"/>
   <addaction name="kelvinAct"/>
   <addaction name="coolAct"/>
   <addaction name="separator"/>
   <addaction name="vignetteAct"/>
   <addaction name="mosaicAct"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="fileOpenAct">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/frontend/resources/pak2/002-open-archive.png</normaloff>:/frontend/resources/pak2/002-open-archive.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="saveImgAct">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/frontend/resources/pak2/003-saving-outlined-diskette-interface-symbol.png</normaloff>:/frontend/resources/pak2/003-saving-outlined-diskette-interface-symbol.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="saveAsAct">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/frontend/resources/pak2/003-saving-outlined-diskette-interface-symbol.png</normaloff>:/frontend/resources/pak2/003-saving-outlined-diskette-interface-symbol.png</iconset>
   </property>
   <property name="text">
    <string>Save As..</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/frontend/resources/pak2/001-close-cross-circular-interface-button.png</normaloff>:/frontend/resources/pak2/001-close-cross-circular-interface-button.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="zoomInAct">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/frontend/resources/pak2/001-increasing-zoom-interface-symbol.png</normaloff>:/frontend/resources/pak2/001-increasing-zoom-interface-symbol.png</iconset>
   </property>
   <property name="text">
    <string>zoom &amp;In (+25%)</string>
   </property>
   <property name="toolTip">
    <string>Zoom in</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="zoomOutAct">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/frontend/resources/pak2/002-decrease-zoom-magnifier-symbol-of-interface-with-minus-sign-inside.png</normaloff>:/frontend/resources/pak2/002-decrease-zoom-magnifier-symbol-of-interface-with-minus-sign-inside.png</iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;Out (-25%)</string>
   </property>
   <property name="toolTip">
    <string>Zoom Out </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="fitToWindowAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/frontend/resources/pak2/001-expand.png</normaloff>:/frontend/resources/pak2/001-expand.png</iconset>
   </property>
   <property name="text">
    <string>FitToWindow</string>
   </property>
   <property name="toolTip">
    <string>Fit Image to Screen</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="normalSizeAct">
   <property name="text">
    <string>&amp;Normal Size</string>
   </property>
   <property name="toolTip">
    <string>Original Image Size</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="grayscaleAct">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/frontend/resources/pak2/001-two-circles-sign-one-black-other-white.png</normaloff>:/frontend/resources/pak2/001-two-circles-sign-one-black-other-white.png</iconset>
   </property>
   <property name="text">
    <string>ToGrayscale</string>
   </property>
   <property name="toolTip">
    <string>Convert to Grayscale</string>
   </property>
  </action>
  <action name="colorizeAct">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/frontend/resources/pak3/002-pixel.png</normaloff>:/frontend/resources/pak3/002-pixel.png</iconset>
   </property>
   <property name="text">
    <string>Colorize</string>
   </property>
   <property name="toolTip">
    <string>Colourize</string>
   </property>
  </action>
  <action name="kelvinAct">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/frontend/resources/pak2/006-thermometer-2.png</normaloff>:/frontend/resources/pak2/006-thermometer-2.png</iconset>
   </property>
   <property name="text">
    <string>KelvinFilter</string>
   </property>
   <property name="toolTip">
    <string>Apply Kelvin Filter</string>
   </property>
  </action>
  <action name="coolAct">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/frontend/resources/pak2/001-snowflake-1.png</normaloff>:/frontend/resources/pak2/001-snowflake-1.png</iconset>
   </property>
   <property name="text">
    <string>CoolFilter</string>
   </property>
   <property name="toolTip">
    <string>Apply Cool Filter</string>
   </property>
  </action>
  <action name="originalAct">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/frontend/resources/pak1/015-photo-camera-2.png</normaloff>:/frontend/resources/pak1/015-photo-camera-2.png</iconset>
   </property>
   <property name="text">
    <string>Original</string>
   </property>
   <property name="toolTip">
    <string>Reset to original</string>
   </property>
  </action>
  <action name="vignetteAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/frontend/resources/pak3/007-vignette.png</normaloff>:/frontend/resources/pak3/007-vignette.png</iconset>
   </property>
   <property name="text">
    <string>Vignette</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="mosaicAct">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/frontend/resources/pak2/002-filter-1.png</normaloff>:/frontend/resources/pak2/002-filter-1.png</iconset>
   </property>
   <property name="text">
    <string>Mosaic</string>
   </property>
   <property name="toolTip">
    <string>Mosaicing</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
